### PR TITLE
add tests for run query endpoint

### PR DESCRIPTION
**Pull request by Leopold-Cramer**

### CODE REVIEW

File: run/src/main/kotlin/com/cosmotech/run/service/RunServiceImpl.kt [hunk 1]
```diff
@@ -296,13 +297,14 @@ class RunServiceImpl(
...
       runDataQuery: RunDataQuery
   ): QueryResult {
-
     checkInternalResultDataServiceConfiguration()
-    getRun(organizationId, workspaceId, runnerId, runId)
+    // getRun(organizationId, workspaceId, runnerId, runId)
 
     val runtimeDS =
         DriverManagerDataSource(
-            "jdbc:postgresql://$host:$port/$runId", readerStorageUsername, readerStoragePassword)
```
Comment (sjoubert) [inline run/src/main/kotlin/com/cosmotech/run/service/RunServiceImpl.kt:305]: Do we go back to hardcode `localhost`, I don't think this is was we want (wrong merge conflict resolve on your part?)

File: run/src/main/kotlin/com/cosmotech/run/service/RunServiceImpl.kt [hunk 2]
```diff
@@ -280,8 +281,8 @@ class RunServiceImpl(
   ): RunData {
 
     checkInternalResultDataServiceConfiguration()
-    val run = getRun(organizationId, workspaceId, runnerId, runId)
```
Comment (sjoubert) [inline run/src/main/kotlin/com/cosmotech/run/service/RunServiceImpl.kt:283]: If this is really not necessary then just remove the code, don't comment it. But I somehow doubt we want to remove RBAC checks. (Although maybe this is done elsewhere now).

Same a few lines below for another `getRun`

File: run/src/main/openapi/run.yaml [hunk 3]
```diff
@@ -753,7 +753,7 @@ components:
           description: the list of results
           items:
             type: object
-            additionalProperties: true
```
Comment (sjoubert) [inline run/src/main/openapi/run.yaml:756]: I'm not sure how an `additionalProperties` to false behave when no specific properties are defined. Is it ok or does it prevent any content (which is probably not what we want)?

**Review by sjoubert — CHANGES_REQUESTED** [review]

**Review by jreynard-code — APPROVED** [review]
LGTM

**Pull request merged**